frontend/src/worksheets/CreateWorkSheet.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { postWorkSheet } from '../api/WorkSheetAPI';
+import { PATH_VARIABLES } from './Const';
+import CreateWorkSheetForm from './CreateWorkSheetForm';
+import {useHistory} from "react-router-dom"
+
+function CreateWorkSheet() {
+  
+  const [sent, setSent] = useState(false);
+  const [sentSuccessfully, setSentSuccessfully] = useState(false);
+  const [popUpMessage, setPopUpMessage] = useState('');
+
+  let history = useHistory();
+
+  function handleClick() {
+    setSent(false);
+    sentSuccessfully 
+    ? history.push(`/${PATH_VARIABLES.BASEPATH_WORKSHEET}`) 
+    : history.push(`/${PATH_VARIABLES.BASEPATH_WORKSHEET}/${PATH_VARIABLES.ENDPATH1_WORKSHEET}`)
```

Comment:
Ilyenkor a /employees/new-n vagyunk? Ha igen, akkor nem is kellene redirect.

Reply:
Igen, töröltem.